specs/android/send_firebase_push_notification.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: Send a firebase push notification to local Android emulator
+command: adb shell am broadcast \
+  -n {{your_app_package_name}}/com.google.firebase.iid.FirebaseInstanceIdReceiver \
+  -a "com.google.android.c2dm.intent.RECEIVE" \
+           --es "title" "{{notification_title}}" \
+           --es "body" "{{notification_body}}" \
+           --es "deeplink" "{{deeplink}}"
+tags:
+  - android
+  - adb
+  - firbase
+  - notification
+description: Uses adb to push a Firebase push notification to your local emulator. Remember that your emulator has to be rooted.You can run `adb root` or the root emulator workflow. Furthermore, you need to implement the firebase provider. This is recomended todo in your debug manifest.
```

Comment:
To avoid making this description too long, I would keep this a little shorter. How about just `Uses adb to push a Firebase push notification to your local emulator`

Reply:
I liked that! Changed it in 9b44064 😄 Thanks!